### PR TITLE
Remove the orchestrator plugin and related code

### PR DESCRIPTION
**Pull request by chmeliik**
# Maintainers will complete the following section

- [x] Commit messages are descriptive enough
- [x] Code coverage from testing does not decrease and new code is covered
- [n/a] Python type annotations added to new code
- [n/a] JSON/YAML configuration changes are updated in the relevant schema
- [n/a] Changes to metadata also update the documentation for the metadata
- [n/a] Pull request has a link to an osbs-docs PR for user documentation updates
- [n/a] New feature can be disabled from a configuration file

### CODE REVIEW

File: atomic_reactor/utils/koji.py [hunk 1]
```diff
@@ -550,9 +550,8 @@ def add_buildroot_id(output: Output) -> Output:
     # Add the 'docker save' image to the output
     image = add_buildroot_id(output)
 
-    # when doing regular build, worker already uploads image,
-    # so orchestrator needs only metadata,
-    # but source contaiener build didn't upload that image yet,
+    # when doing a binary build, the build task already uploads image,
+    # so we need only metadata, but source container build didn't upload that image yet,
     # so we want metadata, and the image to upload
```
Comment (chmeliik) [inline atomic_reactor/utils/koji.py:555]: Pretty sure this is still nonsense, the thing is that we *do* actually seem to upload something extra for source containers. I'll have to look into what that is

Comment (tkdchen) [reply atomic_reactor/utils/koji.py:555]: For uploading a built image koji metadata, koji_import seems generate a "fake" image output file (returned from get_image_output) to represent the built image. From my point of view, these lines of comment is a little useless, and the `get_image_output` function should be described clearly instead, especially what concrete content to be uploaded to the Koji/Brew as part of the metadata.

**Review by tkdchen — COMMENTED** [review]
LGTM :+1:

**Review by rcerven — APPROVED** [review]

File: atomic_reactor/utils/koji.py (hunk 1 above)
Comment (chmeliik) [reply atomic_reactor/utils/koji.py]: Ack, deleted the comment

**Review by tkdchen — APPROVED** [review]

**Pull request merged**